css_tasks/task1/index.html

```diff
@@ -0,0 +1,273 @@
+                                ><i>Italic item</i
+                                ><sup
+                                  >&nbsp;<a href=""
+                                    ><font size="1">2</font></a
+                                  ></sup
+                                ></font
+                              >
+                            </li>
+                            <li>
+                              <font face="Montserrat"
+                                ><strong>Bold item</strong></font
+                              >
+                            </li>
+                          </ul>
+                        </td>
+                      </tr>
+                    </table>
+                  </td>
+                </tr> <!-- end of list items-->
+                <tr>
+                  <td>
+                    <font face="Montserrat"><strong>Items in the shop</strong></font>
+                  </td>
+                </tr>
+                <!-- Inner table -->
+                <tr>
+                  <td>
+                    <table width ="100%" cellpadding ="10px" cellspacing ="0" border="1px">
+                      <thead>
+                        <tr>
+
+                          <th><font face= "Montserrat">N<u><font size="2">o</font></u></font></th>
+
+                          <th><font face= "Montserrat">Name</font></th>
+                          <th><font face= "Montserrat">Price</font></th>
+                        </tr>
+                      </thead>
+                      <tbody>
+                        <tr>
+                          <td><font face ="Montserrat">1</font></td>
+                          <td><font face ="Montserrat">Sword</font></td>
+                          <td><font face ="Montserrat">10</font></td>
+                        </tr>
+                        <tr>
+                          <td><font face ="Montserrat">2</font></td>
+                          <td><font face ="Montserrat">Mace</font></td>
+                          <td><font face ="Montserrat">5</font></td>
+                        </tr>
+                        <tr>
+                          <td><font face ="Montserrat">3</font></td>
+                          <td><font face ="Montserrat">Faithful horse</font></td>
+                          <td><font face ="Montserrat">15</font></td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <br>
+            <br>
+            <br>
+          </td>
+          <!-- Right Sidebar -->
+          <td width="198px" valign="top">
+            <table width="100%">
+              <tbody>
+                <!-- Inputs -->
+                <tr>
+                  <td>
+                    <table width="100%" cellpadding="5px">
+                      <tr>
+                        <td><font face="Montserrat" size="3">Login:</font></td>
```

Comment:
login should be label element associated with input
and all inputs (button as well) should be inside `<form>` element

Reply:
Done it.